group_vars/all/general.yml

```diff
@@ -42,3 +42,16 @@ all_sysctl__to_merge:
 
   # especially on low mem devices this is important
   vm.min_free_kbytes: 1024
+
+
+## Routing Section:
+freifunk_global_prefix: 2001:bf7::/32
+
+# Default mesh metric in inbound direction (rxcost) for normal mesh links
+mesh_metric_default_in: 512
+
+# Default mesh metric in inbound direction (rxcost) for tunnels
+mesh_metric_tunnel_in: 512
```

Comment:
I think in relation to the other metrics where we set ad_hoc to 2048 this default metric is way to low. My understanding was that for nodes with a connection to the mesh we want to have wireless routing preferred and treat tunnels as a failover. With this my default route via the tunnels has a metric of 1024 and the lowest metric via one of the wireless interfaces is 2560 (best case via wireless would be 2048 (5 GHz adhoc) + 128 (60 GHz) + 128 (60 GHz) = 2304).

I think we really need to decide about https://github.com/freifunk-berlin/bbb-configs/issues/969 and should probably base the default on a 50 MBit connection.

Reply:
But from what I can tell the metric for the default route is now lower with birdc:

babel:

```
root@kirchhof-core:~# echo "dump" | nc ::1 33123 | grep 'prefix ::/0'
add route b6e7e610 prefix ::/0 from 2001:bf7::/32 installed yes id f0:83:53:53:82:be:48:d8 metric 1536 refmetric 0 via fe80::1 if ts_wg0
add route b6e7c610 prefix ::/0 from 2001:bf7::/32 installed no id f0:83:53:53:82:be:48:d8 metric 1600 refmetric 64 via fe80::1 if ts_wg1
root@kirchhof-core:~#
```

bird:

```
root@lacasahellersdorf-core:~# birdc show babel route | egrep '^::/0'
::/0 from 2001:bf7::/32                                fe80::1                   ts_wg0      1088 * 21935  55.131
root@lacasahellersdorf-core:~#
```

So looks like a change to me.